chart/README.md

```diff
@@ -541,14 +541,60 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### HPA (Horizontal Pod Autoscaling)
+
+The HPA in Kubernetes automatically scales workload resources based on demand. It deploys more Pods
+for increased load and scales down if the load decreases. Implemented as a Kubernetes API resource and controller, it
+adjusts the target scale periodically based on observed metrics like CPU and memory utilization.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`, and set
+the `hpa.resources.memory.enabled` parameter to `true` to enable memory-based autoscaling.
+
+   ```
+   --set hpa.enabled=true
+   --set hpa.resources.memory.enabled=true
+   ```
+
+Note that HPA requires metrics-server to be installed in the cluster. If you are using EKS, you can install it by
+running the following
+command: `kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml`
+Also, defined resources limits for all the containers in the cluster are required for HPA to be able to properly compute
+metrics.
+
+### Examples
+
+Some example commands are shown below.
+
+   ```
+   helm install --namespace staging mycluster --dependency-update
```

Comment:
should this example have the parameters above for hpa?

Reply:
> commit: [df980a1](https://github.com/sonatype/nexus-iq-server-ha/commit/df980a17278b6efdcf4466bf54dc3e8a3503b7b3)

I'd add another example without the memory parameter, since from what I understand above, it's optional

Reply:
They are not really optional. I mention that defined resources have to be set for HPA to be able to properly compute metrics. 
The optional would be just pass the parameters or update the `values.yml`